src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
-    private bool SyncToPr(uint pr, StringBuilder executionResult)
+    private string GetRepoPath(uint repoId)
```

Comment:
Although we use numbers on CI server as Ids, please don't assume id can only be unsigned number. It's perfectly fine the id is "Tom" or "Mary". We actually already have a special Id that is "Master". Please allow it to be just a string. It will only be valid if you can find "Tom" or Mary" on disk (and you have permission to it); otherwise, it is invalid anyway even if I give you a number.